master/internal/api_experiment.go

```diff
@@ -500,34 +503,63 @@ func (a *apiServer) PatchExperiment(
 	}
 
 	paths := req.UpdateMask.GetPaths()
+	updateNotes := false
 	for _, path := range paths {
 		switch {
-		case path == "description":
-			exp.Description = req.Experiment.Description
+		case path == "name":
+			isEmpty, err := regexp.Match(`^ *$`, []byte(req.Experiment.Name))
```

Comment:
Why not just `if len(strings.TrimSpace(req.Experiment.Name)) == 0`?

Reply:
huh I didn't know about that. 👍🏼